src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.ts

```diff
@@ -11,13 +11,16 @@ import { ArtemisDurationFromSecondsPipe } from 'app/shared/pipes/artemis-duratio
 import { JhiAlertService } from 'ng-jhipster';
 import { round } from 'app/shared/util/utils';
 import * as moment from 'moment';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { ButtonSize } from 'app/shared/components/button.component';
 
 @Component({
     selector: 'jhi-student-exam-detail',
     templateUrl: './student-exam-detail.component.html',
     providers: [ArtemisDurationFromSecondsPipe],
 })
 export class StudentExamDetailComponent implements OnInit {
+    readonly ButtonSize = ButtonSize;
```

Comment:
Is this really needed? I could not found a usage within the html or code :)

Reply:
I removed it